packages/cloud_firestore/cloud_firestore/example/macos/Podfile

```diff
@@ -1,4 +1,9 @@
 platform :osx, '10.12'
+$FirebaseSDKVersion = '7.4.0'
```

Comment:
I removed those in my one version pr, why add one back? Now you have this version to watch and bump even though core is mandatory. Better to remove this and fail fast if can't get version from core then inevitably take version skew or maintain it

Reply:
This isn't the same, this is the overriding sdk versions thing as we need 7.4.0 for macOS support 🙂

Reply:
...instead of bumping firebase_core in the PR?

Reply:
We're in the middle of nullsafety migrations so didn't think it wise to touch anything that affects all packages and potentially cause issues/slow things down, this is meant to be a targeted PR that can be redressed after the nullsafety dust settles by bumping core - I should have left a todo comment 😩

Reply:
Will throw up another PR in bit that reworks this and bumps core, then it can be rebased after nullsafety and merged.

Reply:
That's understandable but a breadcrumb for sure. I will say that the 7.n.0 updates are trivial, if they can't just always be adopted firebase-ios-sdk did something wrong, and they haven't done that for a while. This particular one (7.3.0 to 7.4.0) was just fine in my experience on my work app (already live ;-) )

Reply:
> This particular one (7.3.0 to 7.4.0) was just fine in my experience on my work app (already live ;-) )

Good to know, let's see what happens on the other PR with CI and perhaps we can merge it, maybe I'm worrying too much 😅

Reply:
Actually can't merge yet as it'd cause a feat version bump to all packages which we can't do yet with the nullsafety package versioning prereleases atm - still to be figured out.

I'll get the PR up still anyway.